docs/src/paradox/03-apis/api-v2/editing-resources.md

```diff
@@ -185,6 +185,39 @@ than the requesting user only if the requesting user is an administrator of the
 project or a system administrator. The specified creator must also
 have permission to create resources of that class in that project.
 
+In addition to the creation date, in the body of the request, it is possible to specify a custom IRI for a resource through
+the `@id` attribute which will then be assigned to the resource; otherwise the resource will get a unique random IRI. 
+Similarly, it is possible to assign a custom IRI to the values using their `@id` attributes; if not given, random IRIs
+will be assigned to the values. An optional custom UUID of a value can also be given by adding `knora-api:valueHasUUID`.
+For example:
+```jsonld
+{
+   "@id" : "http://rdfh.ch/0001/a-custom-thing",
+   "@type" : "anything:Thing",
+   "knora-api:attachedToProject" : {
+     "@id" : "http://rdfh.ch/projects/0001"
+   },
+   "anything:hasInteger" : {
+       "@id" : "http://rdfh.ch/0001/a-thing/values/int-value-IRI",
+       "@type" : "knora-api:IntValue",
+       "knora-api:intValueAsInt" : 10,
+       "knora-api:valueHasUUID" : "IN4R19yYR0ygi3K2VEHpUQ"
```

Comment:
We should probably say that the value UUID needs to be a real Base64-encoded UUID, so it is mainly useful for transferring data from one Knora repository to another.

Reply:
I.e. you can't use `"my-custom-UUID"` or any random string like that.